apps/molgenis-components/src/Interfaces/ITableMetaData.ts

```diff
@@ -1,12 +1,14 @@
 import { IColumn } from "./IColumn";
 import { ISetting } from "./ISetting";
+import { ILocale } from "./ILocales";
 
 export interface ITableMetaData {
   columns?: IColumn[];
   name: string;
   tableType: string;
   id: string;
-  description: string;
+  descriptions: [ILocale];
```

Comment:
this indicates an array with exactly 1 ILocale in it, I think you want ILocale[] here to indicate an array of arbitrary length

Reply:
yes silly of me